smart-contracts/Cargo.toml

```diff
@@ -14,9 +14,9 @@ members = [
   "contracts/cl-vault",
   "contracts/merkle-incentives",
   "contracts/range-middleware",
-  "contracts/dex-router-osmosis",
-  "contracts/lst-dex-adapter-osmosis",
-  "contracts/lst-adapter-osmosis",
+  "contracts/cw-vault-multi-standard",
```

Comment:
plz keep old contracts

Reply:
error in the rebase of the toml, readded